src/main/java/de/tum/in/www1/artemis/web/rest/TextSubmissionResource.java

```diff
@@ -296,4 +300,14 @@ else if (!authorizationCheckService.isAtLeastInstructorForExercise(exercise)) {
         }
         return bodyBuilder.body(textSubmission);
     }
+
+    /**
+     * Throws IllegalArgumentException if the text length is over 30000 characters.
+     * @param textSubmission the text submission
+     */
+    private void checkTextLength(TextSubmission textSubmission) {
+        if (textSubmission.getText() != null && textSubmission.getText().toCharArray().length > 30000) {
```

Comment:
Why are we using `toCharArray()` instead of `length()` directly?

Reply:
```suggestion
        if (textSubmission.getText() != null && textSubmission.getText().length > 30000) {
```

Reply:
@FrankeLukas you are right, that is cleaner